examples/vpc-with-dns/outputs.tf

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "vpc_id" {
+  value       = module.slz_vpc.vpc_id
+  description = "VPC id"
+}
+
+output "vpc_crn" {
+  value       = module.slz_vpc.vpc_crn
+  description = "VPC crn"
+}
+
+output "network_acls" {
+  value       = module.slz_vpc.network_acls
+  description = "VPC network ACLs"
+}
+
+output "public_gateways" {
+  value       = module.slz_vpc.public_gateways
+  description = "VPC public gateways"
+}
+
+output "subnet_zone_list" {
+  value       = module.slz_vpc.subnet_zone_list
+  description = "VPC subnet zone list"
+}
+
+output "subnet_detail_map" {
+  value       = module.slz_vpc.subnet_detail_map
+  description = "VPC subnet detail map"
+}
+
+output "dns_zone_state" {
+  description = "The state of the DNS zone."
+  value       = module.slz_vpc.dns_zone_state
+}
+
+output "dns_zone" {
```

Comment:
At the root level, I suggested rendering just `zone_id`. In case you do that, then keep this description, but then modify the `dns_record_ids` description too. 

If you update the output at the root level, then remove `The ID is composed of <instance_id>/<zone_id>`

Reply:
Updated the output at root level to render `zone_id` hence removed the statement mentioned.